voting-bot/database/sqlite_test.go

```
@@ -69,9 +69,12 @@ func TestKeys(t *testing.T) {
 	defer db.Close()
 	_, err = db.Exec("CREATE TABLE IF NOT EXISTS keys (chainName VARCHAR PRIMARY KEY, keyName VARCHAR, keyAddress VARCHAR)")
 	if err != nil {
-		fmt.Println(err)
 		t.Fatalf("Failed to create test table: %v", err)
 	}
+	_, err = db.Exec("ALTER TABLE keys ADD COLUMN authzStatus VARCHAR DEFAULT 'false'")
+	if err != nil {
```

Lets ignore error message If `logs` table already has `authzStatus` column.